modules/admin_manual/pages/enterprise/external_storage/windows-network-drive_configuration.adoc

```diff
@@ -348,14 +288,179 @@ processed.
 As said, you can wrap that command in a Cron job so it’s run every 5
 minutes for example.
 
+=== WND Listener Configuration
+
+Create a service following the instructions below that checks the share for changes:
+
+* For each WND mount point distinguished by a SERVER - SHARE pair,
+** place one copy of a file
+with following content under +
+`/etc/systemd/system/owncloud-wnd-listen-SERVER-SHARE.service`
+** replacing the all upper case words `SERVER`, `SHARE`, `USER` and `PASSWORD`
+** in both, the **filename** and in the **contents** below with their respective values. +
+Take care to also adjust the paths in `WorkingDirectory` and `ExecStart`
+according to your installation.
++
+....
+[Unit]
+Description=ownCloud WND Listener for SERVER SHARE
+After=syslog.target
+After=network.target
+Requires=apache2.service
+[Service]
+User=www-data
+Group=www-data
+WorkingDirectory=/var/www/owncloud
+ExecStart=/usr/bin/php ./occ wnd:listen -vvv SERVER SHARE USER PASSWORD
```

Comment:
There are better ways to set the password for the wnd:listen. The one shown here is a quick and dirty way to set the password, but I don't think we should recommend this

Reply:
Can you then tell me the better ways?

Reply:
@jvillafanez, do you have further feedback?

Reply:
I suspect a variable has to be defined.

Reply:
https://github.com/owncloud/windows_network_drive/pull/112 should contain the relevant information. https://github.com/owncloud/windows_network_drive/pull/129 also contains additional options related to password handling.

Reply:
```
occ wnd:listen host share username --password-file=/tmp/mypass --password-trim
```
probably is the easiest alternative. Just change the file location and make sure that file contains the password.

Reply:
added